Modules/Devices/src/Devices.Infrastructure.Database.Postgres/Migrations/20231121135428_IdentityStatus.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.Modules.Devices.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class IdentityStatus : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_IdentityDeletionProcesses_Identities_IdentityAddress",
+                table: "IdentityDeletionProcesses");
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "DeletionStartedAt",
+                table: "IdentityDeletionProcesses",
+                type: "timestamp with time zone",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
+
+            migrationBuilder.AddColumn<int>(
+                name: "Status",
+                table: "Identities",
+                type: "integer",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_IdentityDeletionProcesses_Identities_IdentityAddress",
+                table: "IdentityDeletionProcesses",
+                column: "IdentityAddress",
+                principalTable: "Identities",
+                principalColumn: "Address",
+                onDelete: ReferentialAction.Cascade);
```

Comment:
Is this intended? Don't you configure it to set NULL on the entity type configuration?

Reply:
It's a great question. If an Identity is deleted on the database level for some reason, this makes all associated identityDeletionProcesses be deleted as well. As far as I know, the only desired remain to be left on the database is the AuditLogEntry(ies), so I don't see how this is an issue.

Reply:
I agree with @Dannyps

Reply:
I have tackled the inconsistency in the latest merge.